LICENSE-README

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2023 See the README.md at the bottom for authors/contributors.
```

Comment:
I'd make this whole file a `.md` file with all the appropriate formatting (headings, links, etc.) to make it easier to read.

Reply:
We could do that, the problem is the longetivity of the .md format. Sounds unrealistic, but though having just a normal text file with no ending would be the best compatible format.
As long as the .md file can be understand correctly and works when read without a Markdown viewer and still works, we could do that.
The LICENSE file could also be an .md file (https://www.gnu.org/licenses/agpl-3.0.md).

Reply:
Files are now `.md`, I made the links, do you have a proposal for heading and so on?